types/babel__traverse/index.d.ts

```diff
@@ -8,30 +8,35 @@
 //                 Ifiok Jr. <https://github.com/ifiokjr>
 //                 ExE Boss <https://github.com/ExE-Boss>
 //                 Daniel Tschinder <https://github.com/danez>
+//                 Avery Warddhana <https://github.com/nullableVoidPtr>
```

Comment:
Is this intentional? Note that we use git history to track contributions; this particular field adds you as an owner of the package and will ask you for reviews, etc.

Reply:
This is one item I forgot to resolve shortly after making this PR. I'll take my name off later today.